docs/advanced/secure-setup.md

```diff
@@ -0,0 +1,372 @@
+    *boot
+    ```
+
+    Edit the config.txt file in *boot
+
+    Find the text [all]
+
+    and add the following lines below it:
+
+    ```no-highlight
+    dtoverlay=disable-wifi
+    dtoverlay=disable-bt
+    ```
+
+    This will disable the WiFi and Bluetooth from being initialized when the system boots. 
+
+You can now copy over `~/copytob.tar.gz` to a writeable volume.
+
+For ease of use, let us make a folder called `bconfig` and copy it over.
+
+(You might need to use an admin or sudo permissions)
+
+eg (These will be different for LiveUSB's)
+`sudo mkdir /media/{yourusername}/writable/bconfig`
+`sudo cp ~/copytob.tar.gz /media/{yourusername}/writable/bconfig`
+
+Now unmount the disk from your system.
+**This will be the last time you will ever connect this disk to a system with internet access.**
+
+Now boot `Wallet-B` with the disk inserted **(DO NOT CONNECT YOUR ETHERNET CABLE)**.
+Depending on the OS it should show you a system setup and a default user creation menu, proceed with a strong password.
+Then once the system is installed it should reboot.
+
+Once logged in get familiar with the system, adjust the clock, etc, and then open a terminal.
+
+??? note "Raspberry Pi specific (click to expand)"
+
+    Optionally you may choose to disable WiFi and Bluetooth at the kernel level too. 
+
+    Create a file in
+
+    ```no-highlight
+    /etc/modprobe.d/raspi-blacklist.conf
+    ```
+
+    and add the contents
+
+    ```no-highlight
+    blacklist brcmfmac
+    blacklist brcmutil
+    blacklist hci_uart
+    blacklist btbcm
+    blacklist btintel
+    blacklist rfcom
+    blacklist btqca
+    blacklist btsdio
+    blacklist bluetooth
+    ```
+
+Disable WiFi and Bluetooth using rfkill:
+
+```no-highlight
+sudo rfkill block wifi
+sudo rfkill block bluetooth
+```
+
+Now to setup the ufw rules run the following commands.
+
+```no-highlight
+sudo ufw default deny outgoing
+sudo ufw default deny incoming
+sudo ufw allow out to {LOCAL_IP_OF_COMPUTER-A} port 9109
+sudo ufw enable
+```
+
+Replace {LOCAL_IP_OF_COMPUTER-A} with the IP address of `Computer-A`
+
+Now you may connect the ethernet cable.
+Wait for the connection to take place and then test if you can connect to `Computer-A` on port 9109
+
+wget {LOCAL_IP_OF_COMPUTER-A}:9109
+
+You should get an error a 400 bad request error. This is fine and shows that a connection is possible.
+
+Now to setup dcrwallet.
+
+First, let us check that the tarfile is unchanged.
+
+`sha256sum /bconfig/copytob.tar.gz`
+
+Once the hash is matched then you can extract it.
+
+```no-highlight
+mkdir ~/decredconfigs
+mkdir ~/decred
+tar -xf /bconfig/copytob.tar.gz -C ~/decredconfigs
+tar -xf ~/decredconfigs/decred-linux-{amd/arm}64-v{{ cliversion }}.tar.gz -C ~/decred
+cp ~/decred/decred-linux-{amd/arm}64-v{{ cliversion }}/* ~/decred/
+rm -rf ~/decred/decred-linux-{amd/arm}64-v{{ cliversion }}/
+```
+
+Now we have the Decred binaries in the ~/decred/decred-linux-{amd/arm}64-v{{ cliversion }} folder.
```

Comment:
Possibly add a line here saying: "Don't run them yet. You need to make the config files first."
Also, do these binaries need to be added to a $PATH$ or something?